src/components/homePage/Footer.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <div class="parent">
+    <div class="footer">
+      <p class="title">CUTC 2021 &mdash; Amplify</p>
+      <p class="contact">Contact us</p>
+      <div class="cutc-icons">
+        <a
+          v-for="(item,index) in items.items"
+          :key="item.link"
+          :href="`${item.link}`"
+          target="_blank"
+          class="icons"
+        >
+          <svg
+            width="40"
+            height="41"
+            viewBox="0 0 40 41"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg"
+            xmlns:xlink="http://www.w3.org/1999/xlink"
+          >
+            <rect y="0.867188" width="40" height="40" :fill="`url(#pattern${index})`" />
+            <defs>
+              <pattern
+                :id="`pattern${index}`"
+                patternContentUnits="objectBoundingBox"
+                width="1"
+                height="1"
+              >
+                <use :xlink:href="`#image${index}`" transform="scale(0.015625)" />
+              </pattern>
+              <image :id="`image${index}`" width="64" height="64" :xlink:href="`${item.image}`" />
+            </defs>
+          </svg>
+        </a>
+      </div>
+      <p>&copy; Copyright 2021 CUTC</p>
+    </div>
+  </div>
+</template>
+
+<script>
+import Items from "../../content/footer-info.json";
```

Comment:
Hmm my thought process for doing this was so that I could separate the presentational logic and application logic. Since I felt like the links and images were not presentational, I kept this data separate. But I could make changes if you feel like this would cause problems or add unnecessary complexity.

Reply:
This is the right thought process, but image data should be stored in image files, not jsons. Please create svg image files in the assets folder as I mentioned :). 

For separating the presentational from the application, I agree with you, but I'd prefer not to clutter up the content folder with files that will not change. Please put the JSON data directly into the state (`data`) of the vue component.

Reply:
This was added in the third commit of this PR, turns out I needed to export instead of copy on the right side of the tree when I clicked on the respective icon